XenAdmin/Actions/OVFActions/ExportApplianceAction.cs

```diff
@@ -92,6 +92,9 @@ protected override void Run()
 		{
 		    SafeToExit = false;
 			var session = Connection.Session;
+
+            InitialiseTicker();

```

Comment:
Personally I'd rather the child classes didn't have to call the ticker, e.g. I would write in the parent class:
```
protected override void Run()
{
    SafeToExit = false;
    InitialiseTicker();
}
```
and in the child class
```
protected override void Run()
{
    base.Run();
    everythingelse();
}
```